lib/psci/psci_suspend.c

```diff
@@ -180,7 +188,8 @@ void psci_cpu_suspend_start(entry_point_info_t *ep,
 	 * Plat. management: Allow the platform to perform the
 	 * necessary actions to turn off this cpu e.g. set the
 	 * platform defined mailbox with the psci entrypoint,
-	 * program the power controller etc.
+	 * program the power controller etc. This handler executes
+	 * with data cache disabled.
```

Comment:
That is only true for `HW_ASSISTED_COHERENCY = 0` platforms. I don't think this comment needs changing but if you do change it, this needs clarifying.